src/exporters/tests/utils/graphql_schema.ts

```diff
@@ -0,0 +1,62 @@
+import { gql, ApolloServer, makeExecutableSchema } from 'apollo-server';
+import { createTestClient } from 'apollo-server-testing';
+import { instrospectionQueryString } from '../../introspection_query';
+
+const typeDefs = gql`
+type Query {
+  rootQuery1(var1: String!): QueryResponse!
+  rootQuery2: QueryResponse!
+}
+
+type Mutation {
+  rootMutation1(var2: Int!, var3: [String!]! = "Default String"): MutationResponse!
+}
+
+type QueryResponse {
+  id: ID!
+  array: [String!]!
+  relation: Object1
+}
+
+type MutationResponse {
+  success: Boolean!
+  code: Int!
+  errors: [String]!
+}
+
+type Object1 {
+  id: ID!
+  object2: Object2
+}
+
+type Object2 {
+  id: ID!
+  object1: Object1!
+}
+`;
+
+const schema = makeExecutableSchema({ typeDefs });
+
+const mocks = {
+  ID: () => 'b1077c17-b48e-4a8d-8b0e-9a09e6352341',
+};
+
+const constructTestServer = () => new ApolloServer({ schema, mocks })
+
+
+// Run this function everytime a change is made to the typeDefs
```

Comment:
The idea here (thinking about the potential evolution of the converters and their tests) is to use the [type definitions above](https://github.com/oshosanya/graphql-export/pull/15/files#diff-33677ed53ca4a6fb553e48de977ecfb5ec244e97f54e3d161957647a2c713767R5) to generate a mock schema and its respective introspection query result, and then [feed that result to the coverters](https://github.com/oshosanya/graphql-export/pull/15/files#diff-1c7eee21a780d6f7e96bd5e516f38c945d7117b230419f376b6545c269b3d387R8)